examples/online-application-portal/response_err_interceptor.bal

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/constraint;
+import ballerina/http;
+
+service class ResponseErrorInterceptor {
+    *http:ResponseErrorInterceptor;
+
+    remote function interceptResponseError(http:RequestContext ctx, error err) returns ClientError|ServerError|error {
+        string path = check ctx.getWithType("rawPath");
```

Comment:
Where do you set these properties to the request context? Without setting these values these wont be there in the default request context

You can use the `http:Request` as a function parameter and get the rawpath and method from the request object

Reply:
Fixed with [db544a94b3468da05aaf66f3837c4c0a861f6358](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/pull/13/commits/db544a94b3468da05aaf66f3837c4c0a861f6358)